Assets/Scripts/FlatSiteCreation.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Mapbox.Unity.Map;
+using Mapbox.Utils;
+
+using RSG;
+using Proyecto26;
+using Models.SteelConnect;
+
+// Type alias for clarity.
+using SiteID = System.String;
+
+public class FlatSiteCreation : MonoBehaviour {
+
+    public GameObject siteMarkerPrefab;
+    public GameObject lineMarkerPrefab;
+
+    public AbstractMap map;
+
+    // Use this for initialization
+    void Start () {
+		
+	}
```

Comment:
Weird indentation but that's probably partly because somehow there's a mix of tab and space indentation here.